bluemira/utilities/positioning.py

```diff
@@ -173,14 +172,17 @@ def _check_geometry_feasibility(self, geometry: BluemiraWire):
         if not self.geometry.is_closed:
             raise PositionerError("RegionInterpolator can only handle closed wires.")
 
-        xz_coordinates = self._get_xz_coordinates()
+        xz_coordinates = self._get_xz_coordinates(10000)
         hull = ConvexHull(xz_coordinates.T)
         # Yes, the "area" of a 2-D scipy ConvexHull is its perimeter...
-        if not np.allclose(hull.area, geometry.length, atol=EPS):
+        if not np.allclose(hull.area, geometry.length, rtol=1e-4, atol=0):
```

Comment:
Is this level of discretisation needed or could we do:
```python
if not np.allclose(hull.area, geometry.length, atol=D_TOLERANCE)
```
obviously needs a bit of further discussion is we talked about earlier

Reply:
Resolved in the next commit, used a more lenient  as discussed.